src/base1/Makefile.am

```diff
@@ -26,7 +26,7 @@ AM_TESTS_ENVIRONMENT = export G_DEBUG=fatal-criticals,fatal-warnings ; export XD
 dist/base1/cockpit.min.css: src/base1/cockpit.css
 	$(AM_V_GEN) $(MKDIR_P) $(dir $@) && \
 	$(srcdir)/tools/missing $(CLEAN_CSS) --keep-line-breaks --output=$@ --inline none --source-map --source-map-inline-sources $^
-dist/base1/patternfly.min.css: dist/base1/patternfly-base.css dist/base1/patternfly-additions.css
+dist/base1/patternfly.min.css: dist/base1/patternfly-base.css
 	$(MIN_CSS_RULE)
```

Comment:
If you do that approach, then just drop that rule, rename the target below, and call MIN_CSS. There's no point in two rules for the same file now.

Reply:
I was not able to do that.
This breaks everything:
``` diff
$ git diff
diff --git src/base1/Makefile.am src/base1/Makefile.am
index b7031cbba..57e4adbdc 100644
--- src/base1/Makefile.am
+++ src/base1/Makefile.am
@@ -26,8 +26,6 @@ AM_TESTS_ENVIRONMENT = export G_DEBUG=fatal-criticals,fatal-warnings ; export XD
 dist/base1/cockpit.min.css: src/base1/cockpit.css
        $(AM_V_GEN) $(MKDIR_P) $(dir $@) && \
        $(srcdir)/tools/missing $(CLEAN_CSS) --keep-line-breaks --output=$@ --inline none --source-map --source-map-inline-sources $^
-dist/base1/patternfly.min.css: dist/base1/patternfly-base.css
-       $(MIN_CSS_RULE)
 dist/base1/jquery.js:
        $(AM_V_GEN) $(MKDIR_P) $(dir $@) && \
        cat $(srcdir)/node_modules/jquery/dist/jquery.js $(srcdir)/node_modules/bootstrap/dist/js/bootstrap.js $(srcdir)/node_modules/patternfly/dist/js/patternfly.js > $@.tmp && $(MV) $@.tmp $@
@@ -67,7 +65,7 @@ dist/base1/react-core-base.css:
 
 # PatternFly 3 with hacked CSS to look like PF4
 dist/base1/patternfly.css.map: dist/base1/patternfly-base.css
-dist/base1/patternfly-base.css: src/base1/patternfly-overrides.less dist/base1/react-core-base.css
+dist/base1/patternfly.min.css: src/base1/patternfly-overrides.less dist/base1/react-core-base.css
        $(AM_V_GEN) $(MKDIR_P) $(dir $@) && \
        F=$(srcdir)/node_modules/patternfly/dist/less/patternfly.less && \
        F_EXTRA=$(srcdir)/node_modules/patternfly/dist/less/patternfly-additions.less && \
@@ -77,7 +75,8 @@ dist/base1/patternfly-base.css: src/base1/patternfly-overrides.less dist/base1/r
        $(LESSC) --source-map=dist/base1/patternfly.css.map --source-map-include-source $$F.tmp $@.tmp && \
        sed -i -f $(srcdir)/tools/patternfly.sed $@.tmp && \
        rm $$F.tmp $$F.patched && \
-       $(MV) $@.tmp $@
+       $(MV) $@.tmp $@ && \
+       $(MIN_CSS_RULE)
 
 dist/base1/fonts/fontawesome.woff:
        $(V_COPY) $(MKDIR_P) $(dir $@) && \
@@ -206,7 +205,6 @@ EXTRA_DIST += \
        dist/base1/mustache.js \
        dist/base1/mustache.min.js \
        dist/base1/mustache.min.js.map \
-       dist/base1/patternfly-base.css \
        dist/base1/patternfly.min.css \
        dist/base1/patternfly.min.css.map \
        dist/base1/patternfly.css.map \
@@ -247,7 +245,6 @@ MAINTAINERCLEANFILES += \
        dist/base1/mustache.min.js \
        dist/base1/mustache.min.js.gz \
        dist/base1/mustache.min.js.map \
-       dist/base1/patternfly-base.css \
        dist/base1/patternfly.min.css \
        dist/base1/patternfly.min.css.gz \
        dist/base1/patternfly.min.css.map \

```

Reply:
Right, because `$MIN_CSS_RULE` takes the dependencies as input. But nevermind, we can clean this up in a subsequent step, it's just a minor wart.